### ISSUE TITLE
Have to delete the .vscode-server/ directory every time i open vscode with ssh extension to connect to a server

### ISSUE THREAD

**Issue by r7butler** [microsoft/vscode#109096]
This issue may need to be reported to the dev of the ssh extension (Remote - SSH 0.49.0)
But it looks like this extension was also issued by microsoft
This is the one I'm talking about

![Screen Shot 2020-10-21 at 7 48 34 AM](https://user-images.githubusercontent.com/40872002/96736797-d3c3f880-1371-11eb-942b-416092790526.png)

The server I connect to with the ssh extension is running Debian 9 (stretch)
The Linux kernel verison is 4.9.0-6-amd64

More precisely, this is what I get when I run cat /proc/version
Linux version 4.9.0-6-amd64 (debian-kernel@lists.debian.org) (gcc version 6.3.0 20170516 (Debian 6.3.0-18+deb9u1) ) #1 SMP Debian 4.9.82-1+deb9u3 (2018-03-02)

- VSCode Version: 1.42.1
- OS Version: macOS Catalina 10.15.7

Steps to Reproduce:

1. set up a docker container that runs Debian 9 with port 22 open
2. set up an ssh server in the container
3. Try and connect to the container with vscode ssh extension
4. Close the session
5. Try to connect again

Basically what I am noticing is that the .vscode-server/ directory that gets created every time i connect, needs to not be there when i connect (Hopefully that made sense)

If it's there, it keeps retrying and failing, so what i have to do is ssh to the server from the terminal, and run:
$ rm -rf ~/.vscode-server

After I do that, when i try to connect to the server via the vscode ssh extension, it works

Does this issue occur when all extensions are disabled?: Yes/No
